packages/emf/backend/sirius-components-emf-forms/src/main/java/org/eclipse/sirius/components/emf/forms/NumberIfDescriptionProvider.java

```diff
@@ -115,7 +124,9 @@ private Function<VariableManager, String> getValueProvider() {
                 EAttribute eAttribute = optionalEAttribute.get();
 
                 Object value = eObject.eGet(eAttribute);
-                return EcoreUtil.convertToString(this.eDataType, value);
+                if (value != null && !eAttribute.isMany()) {
```

Comment:
If we ignore multi-valued attribute (which seem like a good idea), shouldn't this be filtered in the `getPredicate`?